### ISSUE TITLE
Flatpak install exits upon loading image.

### ISSUE THREAD

**Issue by garrettkajmowicz** [AlizaMedicalImaging/AlizaMS#67]
I have a recent set of images in DICOM provided to me by my doctor. I'm attempting to load them.
I am using the stable version from the FlatPak repository. (1.9.10 in About menu)

Despite enabling every debug setting and workaround in the settings menu, I continually get the following:
I navigate to any of the DICOM images stored on the provided disk.
I attempt to load the image.
After a few seconds, the image seems to display on my screen.
The application immediately exits, taking down any possible image view.
No log records are printed to the console after starting, providing no context as to possible causes.

I expected to either view the relevant image(s) or to get a meaningful error/log report which I could provide here.

**Comment by issakomi** [comment]
Thank you for the report. It looks like an application crash. Actually I don't know about the scenario that causes a crash and there are no crash reports. If you could provide me with an image, I will certainly investigate.

**Comment by issakomi** [comment]
If couldn't provide the image there are not many options for me to investigate, you could try following:

 - disable accelerated 3D in 'Settings'-'3D' and restart
 - reset settings to default, s. 'Settings' tab
 - try another package, preferably  [rpm CentOS 8, Red Hat Enterprise Linux 8, Fedora 28 or newer](https://github.com/AlizaMedicalImaging/AlizaMS/releases/download/v1.9.10/alizams-1.9.10-1.x86_64.rpm) or [Single binary](https://github.com/AlizaMedicalImaging/AlizaMS/releases/download/v1.9.10/alizams_binary_only-1.9.10.tar.gz), the last one will require manual install of Qt5 dependencies
 - disable 'Load DICOM ... in separate thread' in 'Settings'->'Application' (very unlikely)

Also it would be interesting to know does it happen with another files. The fact that you can view an image for some time before crash could mean that the problem may be not related to loading of DICOM, but I am not sure.

**Comment by Chaikney** [comment]
I have the same "load image then crash" problem that is reported here. Will copy details before for reference and try what you  suggest above.
Operating System: KDE neon 6.0
KDE Plasma Version: 6.1.5
KDE Frameworks Version: 6.6.0
Qt Version: 6.7.2
Kernel Version: 5.15.0-122-generic (64-bit)
Graphics Platform: Wayland
Processors: 16 × AMD Ryzen 7 5700U with Radeon Graphics
Memory: 7,1 GiB of RAM
Graphics Processor: AMD Radeon Graphics
Manufacturer: SLIMBOOK
Product Name: PROX-AMD5

**Comment by Chaikney** [comment]
None of the things suggested above made any difference to me. Here is the About screen of the version I'm running (couldn't copy it, sorry)
![Screenshot_20240926_192100](https://github.com/user-attachments/assets/956e1a9f-c930-4f3f-bf7e-25286d66a42f)

**Comment by issakomi** [comment]
Thank you very much for the report

> The native package from OBS worked.

Do you mean one of my packages (in Release section) built with Open Build System? If yes, may be you could try [Single binary](https://github.com/AlizaMedicalImaging/AlizaMS/releases/download/v1.9.10/alizams_binary_only-1.9.10.tar.gz), the dependencies should be already installed. Does it work? Thanks.

**Comment by Chaikney** [comment]
> > The native package from OBS worked.
> 
> Do you mean one of my packages (in Release section) built with Open Build System? 

Yes, I think it was the Ubuntu 22.04 one (closest match to Neon I believe)

> If yes, maybe you could try [Single binary](https://github.com/AlizaMedicalImaging/AlizaMS/releases/download/v1.9.10/alizams_binary_only-1.9.10.tar.gz),

I will try this over the weekend, time willing. Feel free to nudge me if you don't hear back in a couple of weeks.

The platform wayland thing will be interesting to try, as that didn't work with the OBS or flatpak package (not to my  surprise).

**Comment by Chaikney** [comment]
I have now tried the single binary you pointed to, and it works. There are no crashes when opening new images.
I tested this using both the default platform `./alizms` and wayland `./alizams -platform wayland`.
The only noticeable difference between the 2 was the icon in the top left corner. (Perhaps the wayland version was faster to launch but since I tried that 2nd, unlikely to be a real effect.)

Not sure if this gets us closer to fixing the flatpak, but it seems like a useable workaround to point people to in the meantime.

**Comment by issakomi** [comment]
Thank you!

> The only noticeable difference between the 2 was the icon in the top left corner. 

On my computer I see a difference if I rotate an image in 3D view, probably something with mouse events (events compression bug? not sure), there is the workaround, check 'Settings' -> '3D' -> 'Set min. update interval'. Just FYI. If X (Xwayland) works, I still prefer X.

I  tried the latest stable KDE neon in VMware, there was an unexpected crash of flatpak release one time, the app just disappeared during a folder scan. Only once, but it is enough to know that something is wrong. I shall look closer and maybe update flatpak build, not yet sure.
